flang/include/flang/Optimizer/Builder/Runtime/Command.h

```diff
@@ -0,0 +1,26 @@
+//===-- Command.h - generate assignment runtime API calls --------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef FORTRAN_OPTIMIZER_BUILDER_RUNTIME_COMMAND_H
+#define FORTRAN_OPTIMIZER_BUILDER_RUNTIME_COMMAND_H
+
+namespace mlir {
+class Value;
```

Comment:
Do you actually use mlir::Value in the header?

Reply:
Yeah, it is used as the return variable for the genArgumentCount function below.
https://github.com/flang-compiler/f18-llvm-project/blob/d11a899092cf853fd95a2be12b77d2cb786b0a28/flang/include/flang/Optimizer/Builder/Runtime/Command.h#L23